specification/monitor/resource-manager/microsoft.insights/stable/2018-04-16/scheduledQueryRule_API.json

```diff
@@ -0,0 +1,794 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Insights API",
+    "version": "2018-04-16",
+    "x-ms-code-generation-settings": {
+      "name": "MonitorManagementClient"
+    },
+    "description": "Azure Monitor client to create/update/delete Scheduled Query Rules"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.insights/scheduledQueryRules/{ruleName}": {
+      "put": {
+        "description": "Creates or updates an log search rule.\r\nRequest method: PUT\t\tRequest URI: https://management.azure.com/subscriptions/{subscription-id}/resourceGroups/{resource-group-name}/providers/microsoft.insights/scheduledQueryRules/{logsearch-rule-name}?api-version={api-version}",
```

Comment:
The `description` should  document the API (the presence of the swagger file is telling you how you should physically call this method). You can drop everything after `Creates or updates an log search rule.`

Reply:
Fixed in latest iteration